cmd/grpc/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	grpc2 "github.com/stasBigunenko/monorepa/pkg/grpc"
```

Comment:
It is generally a bad idea to give to custom packages the same name as basic packages are called, like grpc or jwt. It's better to use "grpcserver", "jwtservice". So that you dont' have to use such names as "grpc2" :)